### PR TITLE
Upgrading base and ml-dev images to standardize on JupyterLab

### PR DESCRIPTION
**Pull request by resdntalien**
# Overview

We have standardized on JupyterLab in the cookiecutter. But there is some missing functionality in the base image that makes using JupyterLab a pain still -- mainly we need to have node and npm installed to get the JupyterLab extensions live.  In this PR I update the base-sys image to have node and npm and also the ml-dev image to have the JupyterLab extension capability installed.  I'm installing the JupyterLab extension capability upstream here because it is a VERY lengthy install (~10mins). Doing this over and over in CI for specific projects is painful. It's much better suited to be here. 

# Changes

* Added node.js and npm to the base-sys image
* Added JupyterLab extensions to ml-dev image

### CODE REVIEW

File: orbyter-base-sys/3.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        vim \
+        emacs \
+        screen \
+        git \
+        wget \
+        curl \
+        cmake \
+        htop \
+        rsync \
+        nodejs \
```
Comment (jeffmm) [inline orbyter-base-sys/3.1/Dockerfile:28]: Just FYI, this line is redundant as nodejs is a dependency of npm

File: orbyter-ml-dev/3.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# orbyter-ml-dev:3.0
+FROM manifoldai/orbyter-base-sys:3.0
```
Comment (jeffmm) [inline orbyter-ml-dev/3.1/Dockerfile]: Nevermind

**Review by jeffmm — APPROVED** [review]
`tqdm` is working on this image for me. I have one inline comment but LGTM!

File: orbyter-base-sys/3.1/Dockerfile (hunk 1 above)
Comment (resdntalien) [reply orbyter-base-sys/3.1/Dockerfile:28]: Makes sense. I'm just gonna let it ride and let the dependency manager figure it out. I don't think there's a downside, right?

**Pull request merged**